pulp_container/tests/functional/api/test_pull_content.py

```diff
@@ -139,7 +141,7 @@ def test_api_returns_same_checksum(self):
         # Assert that at least one layer is synced from remote:latest
         # and the checksum matched with remote
         self.assertTrue(
-            any([result["blobSum"] in checksums for result in get_docker_hub_remote_blobsums()]),
+            any([result["blobSum"] in checksums for result in get_blobsums_from_remote_registry()]),
```

Comment:
and here parsing would be changed as well a bit, it will be list of dictionaries
```
 [
        {
            "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
            "size": 32654,
            "digest": "sha256:e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7fc331f"
        },
        {
            "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
            "size": 16724,
            "digest": "sha256:3c3a4604a545cdc127456d94e421cd355bca5b528f4a9c1905b15da2eb4a4c6b"
        },
        {
            "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
            "size": 73109,
            "digest": "sha256:ec4b8955958665577945c89419d1af06b5f7636b4ac3da7f12184802ad867736"
        }
    ]
```